requirements.txt

```diff
@@ -1,28 +1,27 @@
+click
+git+https://github.com/feder-cr/lib_resume_builder_AIHawk.git
+httpx~=0.27.2
+inputimeout==1.0.4
+jsonschema==4.23.0
+jsonschema-specifications==2023.12.1
 langchain==0.2.11
+langchain-anthropic
 langchain-community==0.2.10
 langchain-core===0.2.36
+langchain-google-genai==1.0.10
+langchain-ollama==0.1.3
 langchain-openai==0.1.17
 langchain-text-splitters==0.2.2
 langsmith==0.1.93
 Levenshtein==0.25.1
+loguru==0.7.2
+linkedin-api
 openai==1.37.1
+pdfminer.six==20221105
+pytest>=8.3.3
+python-dotenv~=1.0.1
+PyYAML~=6.0.2
 regex==2024.7.24
 reportlab==4.2.2
 selenium==4.9.1
 webdriver-manager==4.0.2
-click
-git+https://github.com/feder-cr/lib_resume_builder_AIHawk.git
-linkedin-api
-pdfminer.six==20221105
-jsonschema
```

Comment:
```jsonschema``` package was duplicated on line 17 (no version) and line 22 (version```==4.23.0```)...

the specified version remains in the updated file